container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -34,7 +34,22 @@ TEMPEST_ARGS=""
 [[ ! -z ${TEMPESTCONF_FLAVOR_MIN_DISK} ]] && TEMPESTCONF_ARGS+="--flavor-min-disk ${TEMPESTCONF_FLAVOR_MIN_DISK} "
 [[ ! -z ${TEMPESTCONF_NETWORK_ID} ]] && TEMPESTCONF_ARGS+="--network-id ${TEMPESTCONF_NETWORK_ID} "
 
+TEMPESTCONF_OVERRIDES="$(echo ${TEMPESTCONF_OVERRIDES} | tr '\n' ' ') identity.v3_endpoint_type public"
+
 # Tempest arguments
+TEMPEST_EXTERNAL_PLUGIN_GIT_URL="${TEMPEST_EXTERNAL_PLUGIN_GIT_URL:-}"
+TEMPEST_EXTERNAL_PLUGIN_CHANGE_URL="${TEMPEST_EXTERNAL_PLUGIN_CHANGE_URL:-}"
+TEMPEST_EXTERNAL_PLUGIN_REFSPEC="${TEMPEST_EXTERNAL_PLUGIN_REFSPEC:-}"
+TEMPEST_EXTERNAL_PLUGIN_DIR=/var/lib/tempest/external-plugins
+
+# Convert comma separated lists to arrays
+OLD_IFS=$IFS
```

Comment:
just out of curiosity, what does this stand for?

Reply:
IFS is [Internal Field Separator](https://www.baeldung.com/linux/ifs-shell-variable)

Reply:
right! I totally forgot some bash details :/ :D